config.def.h

```diff
@@ -12,6 +12,21 @@ enum {
  */
 
 #endif
+
+#ifdef _TITLE_CONFIG
```

Comment:
I don't think we need to guard around this. I don't think the existence of the feature actually hurts anyone who doesn't want to use it? If we keep them, I'd lean towards making it an opt-out feature. 

Actually upon further inspection, I think I'm missing the point. We define _TITLE_CONFIG in options.c before including so we benefit do we get to guarding this?

Reply:
Just keeps things consistent with the rest of `config.def.h`. If you take another look, you'll notice that EVERYTHING is guarded.

Reply:
This was standard practice from sxiv. The config file is chunked and the `#include`s only use the part they want to.

Reply:
even if this isn't needed it is an artifact of the codebase for the sake of consistency, as such i'm marking it as resolved.

Reply:
Perhaps the consistency point is enough to drop the comment. But I don't understand why everything is guarded. It isn't easily possible to not compile these extra lines and doing so would break the program anyways. They just seem like glorified comments. I just want to make sure I'm not missing some deep reason for their existence.